src/org/labkey/targetedms/query/PeptideManager.java

```diff
@@ -188,6 +189,7 @@ public static List<PeptideCharacteristic> getModifiedPeptideCharacteristics(long
         sql.append(" AND pep.StartIndex IS NOT NULL AND pep.EndIndex IS NOT NULL ");
         sql.append(" GROUP BY pep.Sequence,pci.SampleFileId, pep.PeptideModifiedSequence, pep.StartIndex, pep.EndIndex ) X ");
         sql.append(" GROUP BY X.Sequence, X.PeptideModifiedSequence, X.StartIndex, X.EndIndex ");
+        sql.append(" ORDER BY X.Sequence ");
```

Comment:
But I think that here you want to sort by PeptideModifiedSequence, not Sequence.